JSTasks/Ajax_function_task/form.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <script defer src="register.js"></script>
+    <title>Registration Form</title>
+</head>
+
+<body>
+
+    <form method="POST" onsubmit="return false;">
+        <fieldset class="fieldset-container">
+            
+            <legend> Registration form </legend>
+
+            <div class="input-container">
+                <label for="username">Username:</label>
+                <input type="text" 
+                       id="username"
+                       minlength="3"
+                       maxlength="10"
+                       required
+                       pattern="[\w_]+"
+                       placeholder="Enter a username"
+                       title=" The username must contain 3 to 10 characters (letters, numbers and _ ).">
+                       
+                <lebel id="usernameError"></lebel>
+                 
+            </div>
+            
+            <div class="input-container">
+                <label for="password">Password: </label>
+                <input type="password" 
+                       id="password"
+                       placeholder="Enter a password"
+                       minlength="6"
+                       required
+                       title="The password must be at least 6 characters, including uppercase, lowercase, and number.">
+           
+                <lebel id="passwordError"></lebel>
+  
+            </div>
+
+            <div class="input-container">
+                <label for="validatePassword">Validate password: </label>
+                <input type="password" 
+                       id="validatePassword"
```

Comment:
You may want to rename the id. You could read a bit here: http://bdavidxyz.com/blog/how-to-name-css-classes/ point 2 Currently the name sounds like a name of a function - as if it is an action and it is going to do something. You may want to rename it to something that would read as if referring to an element, e.g. "confirmation-password" or something similar.